spec/models/master_file_spec.rb

```diff
@@ -348,6 +353,19 @@
           Rails.application.secrets.matterhorn_client_media_path = media_path
           expect(subject.working_file_path).to eq(File.join(media_path,original))
         end
+        it "should not disappear after the post_processing_file_management executes" do
+          new_media_object.collection = collection
+          subject.media_object = new_media_object
+          @old_strategy = Settings.master_file_management.strategy
+          @old_path = Settings.master_file_management.path
+          Settings.master_file_management.strategy = 'move-ui-upload-only'
+          Settings.master_file_management.path = media_path
+          subject.send(:post_processing_file_management)
+          subject.reload
+          Settings.master_file_management.strategy = @old_strategy
```

Comment:
Could you please help  me to understand why we'd need to reset the settings here for the tests?```Settings.master_file_management.strategy = @old_strategy
Settings.master_file_management.path= @old_path
```

Reply:
`Settings` has a global scope. There are subsequent tests that rely on the value set in the test config. The tests for the different values `Settings.master_file_management.strategy` could be better formulated -- I didn't go far enough to change existing tests.

Reply:
Ah, okay that makes sense! thanks!